src/plugins/ardura/Scrollscope/1.4.0.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: Scrollscope
+author: Ardura
+homepage: https://github.com/ardura/Scrollscope
+description: A simple scrolling oscilloscope in rust with multichannel inputs and an analyzer
+date: 2024-07-12T07:00:00.000Z
+license: GPL3
+tags:
+  - Effect
```

Comment:
Any other sub categories which would help categorize it?

Reply:
This plugin is for analysis so unfortunately that's the only category it fits under right now